website/integrations/services/sonarr/index.md

```diff
@@ -54,6 +54,8 @@ sonarr_user: username
 sonarr_password: password
 ```
 
+Additional note:  Make sure the type of authentication is set to "Basic (Browser Popup)" in Sonarr, otherwise the credentials don't ge passed.
```

Comment:
```suggestion
Additional note:  Make sure the type of authentication is set to "Basic (Browser Popup)" in Sonarr, otherwise the credentials are not passed.
```